tests/alive-tv/freshbid-alloca.src.ll

```diff
@@ -0,0 +1,10 @@
+define i8 @freshbid_alloca(i8* %ptr0) {
+  %ptr = alloca i8
+  store i8 10, i8* %ptr
+  store i8 20, i8* %ptr0
+  %v = load i8, i8* %ptr
+  ret i8 %v
+}
+
+
+; ERROR: Value mismatch
```

Comment:
I think I'm misunderstanding something here.
You do a store of 10 and then a load from the same ptr. Isn't `%v` guaranteed to be 10 always? Why does the test fail then?
Same comment for the malloc test

Reply:
I am sorry, this was my mistake. The `; ERROR: Value mismatch` shouldn't have been there.
I removed those.